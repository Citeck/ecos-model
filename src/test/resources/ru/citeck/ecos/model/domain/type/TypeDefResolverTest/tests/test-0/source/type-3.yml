---
id: type-3
parentRef: emodel/type@type-2
