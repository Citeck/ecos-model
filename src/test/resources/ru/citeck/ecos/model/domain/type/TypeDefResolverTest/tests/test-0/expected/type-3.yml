---
id: type-3
name:
  en: type-3

formRef: uiserv/form@form-3
parentRef: emodel/type@base

properties:
  type-3-prop: value-3

aspects:
  - ref: emodel/aspect@e112ff4a-db75-499e-b33c-e3809ac067a2
